Update draft-rosomakho-tls-ech-keylogfile.md (#2)
yaroslavros/tls-ech-keylog: draft-rosomakho-tls-ech-keylogfile.md

Implementers MUST take measures to prevent unauthorized access to the SSLKEYLOGFILE text file.

> According to SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in systems where TLS only protects test data. While the access that this information provides to TLS connections can be useful for diagnosing problems while developing systems, this mechanism MUST NOT be used in a production system.

Reviewer: ```suggestion
According to SSLKEYLOGFILE specification {{!I-D.ietf-tls-keylogfile}}, this extension is intended for use in systems where TLS only protects test data. While the access this information provides to TLS connections can be useful for diagnosing problems during development, this mechanism MUST NOT be used in a production environment.
```
